customtypes/partnerspage/index.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "partnerspage",
+  "label": "PartnersPage",
+  "format": "custom",
+  "repeatable": false,
+  "status": true,
+  "json": {
+    "Main": {
+      "title": {
```

Comment:
We might need to add seo_title and seo_description as well. As other pages, this will be used with usePrismicSEO() in partners.vue.

Reply:
I renamed the existing ones to be SEO